src/components/attendee/PortalModal/PortalModal.tsx

```diff
@@ -1,35 +1,73 @@
 import React, { MouseEventHandler } from "react";
-import { Popover, PopoverProps } from "components/attendee/Popover";
+import classNames from "classnames";
+
+import { STRING_DASH_SPACE, STRING_SPACE } from "settings";
 
 import { Room } from "types/rooms";
+import { WorldEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import { formatTimeLocalised } from "utils/time";
 
 import { useSpaceById } from "hooks/spaces/useSpaceById";
+import { useRelatedVenues } from "hooks/useRelatedVenues";
 
 import { Button } from "../Button";
 
 import CN from "./PortalModal.module.scss";
 
-interface PortalModalProps extends PopoverProps {
+interface PortalModalProps {
   onEnter?: MouseEventHandler<HTMLButtonElement>;
   portal: Room;
+  event?: WorldEvent;
 }
 
 export const PortalModal: React.FC<PortalModalProps> = ({
-  referenceElement,
   onEnter,
   portal,
+  event,
 }) => {
   const { space } = useSpaceById({ spaceId: portal.spaceId });
 
+  const portalSpaceId = portal.spaceId;
+
+  const { findVenueInRelatedVenues } = useRelatedVenues();
+
+  const portalVenue = findVenueInRelatedVenues({ spaceId: portalSpaceId });
+
+  const numberOfUsersInRoom = portalVenue?.recentUserCount ?? 0;
+
+  const popupWrapperClasses = classNames(CN.popupWrapper, {
+    [CN.popupWrapperAligned]: !!numberOfUsersInRoom,
+  });
+
   return (
-    <Popover referenceElement={referenceElement}>
```

Comment:
why have you removed Popover component? 
Now there's an issue of portals overlapping each other?

<img width="321" alt="image" src="https://user-images.githubusercontent.com/63313732/162236453-d392d9a3-3d4b-44bc-8f3a-4fa6d55ac3cf.png">

also, we might need to add a scrollbar for the long description. But it's probably, OOS.

Reply:
I've had some issues with `Popover`'s positioning and decided to get rid of it completely
As I can see these modals overlap at the mockup too, so not sure if that's a huge deal https://sparkletown.github.io/sparkle-design-mockups/attendee-alpha-1-flat/image-map.html#!
Although if this is a crucial thing to avoid we might need to reintroduce `Popover` there, but we'd need to work through corner cases too, like what happens if we have too many portals, so that there's no place to put Popover onto not to overlap another portal, something like that
i'll add the scroll tho, it feels like it's necessary

Reply:
Fixed this in [a90f5ad](https://github.com/sparkletown/sparkle/pull/3008/commits/a90f5adacd3cadd972d4f9779f4c69de64eef3f2)
We'll now only be able to open 1 modal at a time, although they would still overlap I believe ( which is not a big deal )